crates/holochain_conductor_api/src/storage_info.rs

```diff
@@ -0,0 +1,26 @@
+use holochain_types::prelude::*;
+
+/// Storage blob used by hApps to store data.
+#[derive(Clone, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize, SerializedBytes)]
+pub struct AppDataStorageBlob {
```

Comment:
Might be more instructive to call this something like `DnaStorageInfo`. Direct users of the Conductor will know what a DNA is and will probably find that more useful. The Launcher may choose to not use the term DNA, but that's a UI concern.

Reply:
Yes that makes sense to me, renamed